### DISCUSSION TITLE
Welcome to PseudoTV Live Discussions!

### DISCUSSION THREAD

**Discussion by Lunatixz** [PseudoTV/PseudoTV_Live#35] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MrC001** [comment]
I used this years ago -- didn't know it was still out there... will try fooling around.. I see the resource repo fails thoug.. will try manual instal....lot of playing to do i guess !! :)

**Comment by vesarious** [comment]
I have been sick for a long time and hoped to get this when it was ready. Now I fear I have missed it and its no longer available. This makes me sad. Is there any way of still getting this for library files. please

**Reply by Lunatixz** [reply]
This project is still active, if you want to install it download the repository....

**Comment by vesarious** [comment]
Thank You.  I am so happy I cant describe it.  I got it installed and Im trying to figure it out. I works well but there is lagging between changes and I havent figured out how to make channels. So far it out found everything. I need to learn how to create proper channels so I will study. Havent found all the info yet but I can did and figure things out. I also have to step up and help with info to you. I have every intention of donating as well. You truly made my day.
The reason this is so important is that Im an old guy.. I live around poor old people and often they have nothing and would not get service in hospital or in care if they get in trouble and they suffer.  This will provide comfort and help them during their visit in care.  Its my way of trying to support them.   You are truly a brilliant man Thank You.
________________________________

**Comment by kayaaron13** [comment]
Hello i was just wanting to let you know that ive been intrigued and obsessed with this project for years. Its probably my favourite thing about Kodi. I love taking the time to curate and create my own personal viewing experience.  I even like troubleshooting lol. Thank you for the time you have put into this. You are appreciated.

**Comment by blackboxln2** [comment]
Similar thoughts to kayaaron. I honestly enjoy the building part more than the viewing part. I've been on a mission to replicate the ultimate version of my childhood cable experience down to which numbers were assigned. NBC was 5, Nickelodeon is 34, etc. I've written up numerous spreadsheets of how to get closest to this while allowing for channels that no longer exist or have changed names and/or content. I'm also finally letting myself dive into programing/electronics/3d printing type stuff after refusing to follow through with these interests for almost 40 years. Things like PsuedoTV are a great way for me to tread lightly into this world while learning new stuff at my own pace. Ultimately I'd love to know exactly how each component of this project works and why. I do think I'll get there!
Thank you for the years of work and passion for such a niche idea.
